### DISCUSSION TITLE
Meeting Info Doesn't Publish

### DISCUSSION THREAD

**Discussion by dennisguilbault** [code4recovery/12-step-meeting-list#983] [Q&A] [answered]
For some reason, all of a sudden, I can't add information to my meetings and either publish or preview.

**Comment by kiyote33** [comment]
Hi @dennisguilbault,

Can you add some detail about your situation?  The scenario you describe usually only happens when partial edits take place, such as changing an online meeting to in-person without ensuring there is a valid address. Need a bit more detail about what type of edit you are trying to perform so we can try and replicate your situation.

**Comment by dennisguilbault** [comment]
I'm having a doozy of a time here.  It seems that I can't make ANY
changes.  I'm not sure what detail you want.  There is no issue with the
address. (or at least none that I can see).

On Tue, Dec 6, 2022 at 1:07 PM Wayne Parker ***@***.***>
wrote:

> Hi @dennisguilbault <https://github.com/dennisguilbault>,
>
> Can you add some detail about your situation? The scenario you describe
> usually only happens when partial edits take place, such as changing an
> online meeting to in-person without ensuring there is a valid address. Need
> a bit more detail about what type of edit you are trying to perform so we
> can try and replicate your situation.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/983#discussioncomment-4326248>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKR3OHLPEOO3O3FO747DWL6FF5ANCNFSM6AAAAAASUU7UR4>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/983/comments/4326248@
> github.com>
>


-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Comment by dennisguilbault** [comment]
Well, now it works if i log out and log back in.  Very strange.

On Tue, Dec 6, 2022 at 1:07 PM Wayne Parker ***@***.***>
wrote:

> Hi @dennisguilbault <https://github.com/dennisguilbault>,
>
> Can you add some detail about your situation? The scenario you describe
> usually only happens when partial edits take place, such as changing an
> online meeting to in-person without ensuring there is a valid address. Need
> a bit more detail about what type of edit you are trying to perform so we
> can try and replicate your situation.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/983#discussioncomment-4326248>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKR3OHLPEOO3O3FO747DWL6FF5ANCNFSM6AAAAAASUU7UR4>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/983/comments/4326248@
> github.com>
>


-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Reply by kiyote33** [reply]
Hmm. Sort of like the old "Reboot the OS" when nothing else makes sense. If it happens again, try to note anything peculiar that may have taken place and pass it on. Sometimes adding a plugin or making changes to your theme can have unexpected effects on other plugins.

**Comment by dennisguilbault** [comment]
I'm stuck.  Nothing has changed, but when i go into a meeting to edit it
the Update button is not active.

On Tue, Dec 6, 2022 at 2:32 PM Wayne Parker ***@***.***>
wrote:

> Hmm. Sort of like the old "Reboot the OS" when nothing else makes sense.
> If it happens again, try to note anything peculiar that may have taken
> place and pass it on. Sometimes adding a plugin or making changes to your
> theme can have unexpected effects on other plugins.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/983#discussioncomment-4326788>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKR2FUZD2MKMRTYQ6ULTWL6PGVANCNFSM6AAAAAASUU7UR4>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/983/comments/4326788@
> github.com>
>


-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Comment by dennisguilbault** [comment]
Just one more odd thing.  None of my maps are working in the editor.  They do show when i show the website.

**Reply by joshreisner** [reply]
hi @dennisguilbault could you [open your javascript console](https://www.coursera.support/s/article/learner-000001653-How-to-open-the-Javascript-console?language=en_US) on the meeting edit page and send us a screenshot? for example:

<img width="1000" alt="console" src="https://user-images.githubusercontent.com/1551689/206032807-f3883e31-00b1-40f1-a5e1-2ce9853074fc.png">

**Comment by dennisguilbault** [comment]
[image: image.png]

On Tue, Dec 6, 2022 at 4:02 PM Josh Reisner ***@***.***>
wrote:

> hi @dennisguilbault <https://github.com/dennisguilbault> could you open
> your javascript console
> <https://www.coursera.support/s/article/learner-000001653-How-to-open-the-Javascript-console?language=en_US>
> on the meeting edit page and send us a screenshot? for example:
>
> [image: console]
> <https://user-images.githubusercontent.com/1551689/206032807-f3883e31-00b1-40f1-a5e1-2ce9853074fc.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/983#discussioncomment-4327324>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKR2QCDLIMOIQV7FWL63WL6ZYNANCNFSM6AAAAAASUU7UR4>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/983/comments/4327324@
> github.com>
>


-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Reply by joshreisner** [reply]
hi @dennisguilbault - it didn't come through, i don't think email attachments work. could you go to github to upload the image?

**Comment by dennisguilbault** [comment]
![Screenshot 2022-12-06 162348](https://user-images.githubusercontent.com/94577991/206037015-e874b308-9bdd-43f1-a440-49c323efe1f0.jpg)

**Reply by joshreisner** [reply]
terrific, thanks @dennisguilbault - this is another manifestation of the issue we discussed recently at https://github.com/code4recovery/12-step-meeting-list/discussions/590

in this session you are logged in to the site at http://www.aatopeka.org but the site thinks its address should be https://www.aatopeka.org, which is preventing the page from working properly

if it's working in edge, my guess is you were able to log in properly to the https version of the site.

somehow what you want to have happen is to never use http:// and only use https://

**Comment by dennisguilbault** [comment]
VERY INTERESTING ... i tried on Microsoft Edge and it works fine.

**Comment by dennisguilbault** [comment]
and even more interesting ... i don't have to manually refresh the page when i change filters

**Comment by dennisguilbault** [comment]
yeah but when i tried to change it to https it didn't work at all!  what am
i doing wrong?

On Tue, Dec 6, 2022 at 4:36 PM Josh Reisner ***@***.***>
wrote:

> terrific, thanks @dennisguilbault <https://github.com/dennisguilbault> -
> this is another manifestation of the issue we discussed recently at #590
> <https://github.com/code4recovery/12-step-meeting-list/discussions/590>
>
> in this session you are logged in to the site at http://www.aatopeka.org
> but the site thinks its address should be https://www.aatopeka.org, which
> is preventing the page from working properly
>
> if it's working in edge, my guess is you were able to log in properly to
> the https version of the site.
>
> somehow what you want to have happen is to never use http:// and only use
> https://
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/983#discussioncomment-4327525>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKRYJLOQKVMXY3RGNWALWL65YDANCNFSM6AAAAAASUU7UR4>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/983/comments/4327525@
> github.com>
>


-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Reply by joshreisner** [reply]
i don't know - it's a website issue rather than an issue with the plugin. perhaps your hosting company can recommend a way to force SSL for all users?

**Comment by dennisguilbault** [comment, marked as answer]
ok i uploaded and installed the WP Force SSL & HTTPS SSL Redirect plugin and now it's working fine.  I guess i missed this before.
Sorry for any inconvenience.

**Reply by joshreisner** [reply]
excellent!!
